### ISSUE TITLE
Use ssh urls for github remotes?

### ISSUE THREAD

**Issue by wanderingstan** [OriginProtocol/origin-box#32]
When attempting to `git push` some changes to `origin-js`, I was prompted for git username and password, even though I have my ssh key set for my github account.

Turns out we have `https` URLs for our remote
```
$ git config remote.origin.url
remote.origin.url=https://github.com/OriginProtocol/origin-js.git
```

The "fix" for no-prompt github interactions is to change to an ssh url: 
```
git remote set-url origin git@github.com:OriginProtocol/origin-js.git
```

Can we change `origin-box` to use ssh URL's by default when cloning repos? Or would this cause problems for casual developers?

**Comment by tomlinton** [comment]
Thanks for bringing this up @wanderingstan, it was annoying me too. I think we are pretty safe to change it over to SSH by default. Most people who will push will have their github account setup with a SSH key. I'll put a PR up for it.

**Comment by wanderingstan** [comment]
Your work on this is amazing, Tom!

**Comment by franckc** [comment]
Implemented in #33
